server/api/submission/submission.model.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var mongoose = require('bluebird').promisifyAll(require('mongoose'));
+var Schema = mongoose.Schema;
+
+var SubmissionSchema = new Schema({
+  title: String,
+  content: String,
+  images: [String], // path to image on static image server?
+  time: Date,
+  authors: [{type : Schema.Types.ObjectId, ref: 'User'}],
```

Comment:
Should distinguish user who submitted it from people who are claimed to be in the image (those checking in)